notebooks/official/tabnet/ai-explanations-tabnet-algorithm.ipynb

```diff
@@ -0,0 +1,620 @@
+{
```

Comment:
<p>Links are incorrect.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/732/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>I updated the links.</p>

Reply:
<p>Link address is: https://colab.research.google.com/github/GoogleCloudPlatform/vertex-ai-samples/blob/master/notebooks/official/tabnet/ai-explanations-tabnet-algorithm.ipynb.ipynb</p><p> </p><p>Should be https://colab.research.google.com/github/GoogleCloudPlatform/vertex-ai-samples/blob/master/notebooks/official/tabnet/ai-explanations-tabnet-algorithm.ipynb</p>

Reply:
Updated. Thanks.

Reply:
<p>Are you able to open these links yourself or do they only work once the pull request is accepted? What do I need to do to actually go through the notebook steps?</p>

Reply:
No, the link is only ready when the notebook is checked in.
You can use this link to try it: https://colab.sandbox.google.com/github/longtle/vertex-ai-samples/blob/tabnet/notebooks/official/tabnet/ai-explanations-tabnet-algorithm.ipynb